src/components/TableCell.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Link from 'redux-first-router-link';
+
+const TableCell = ({
+  entry = {},
+  field = {},
+}) => (
+  <td data-label={field.label}>
+    {entry.linkType
+      ? <Link to={{ type: entry.linkType, payload: entry.linkPayload }}>{entry.value}</Link>
```

Comment:
We could use `entry.link = {}` to define type and payload directly on the object.

Reply:
That would indeed be nice – but some of the link values are determined by ternary statements, so this seems to be a good middle ground between cluttering the selector and cluttering the component.